src/doc/unstable-book/src/language-features/const-limit.md

```diff
@@ -0,0 +1,7 @@
+# `const_limit`
```

Comment:
Bikeshed: I would prefer `const_eval_limit`. cc @Centril

Reply:
I was leaning towards `const_eval_limit` too, since it is more expressive.
But decided against it, since it was a request to keep it more arbitrary

Reply:
This is easily changeable based on community & language team discussion so go with whatever makes the PR easiest to land for now. ^^

I prefer `const_limit` since we don't have to care about e.g. "interpretation vs. evaluation vs. reduction vs. ..." and because it's shorter... but this is probably for an RFC to discuss & lay out the pros & cons of various alternatives.